### ISSUE TITLE
missing option names

### ISSUE THREAD

**Issue by ciechompl** [perara/ha_systemair#2]
HA 0.111.3
When adding integration, options titles are missing : 

![image](https://user-images.githubusercontent.com/27088154/84828828-f8583e80-b026-11ea-8761-86e4aa13aff8.png)

I guess that first one is IAM ID, second PASSWORD :)

**Comment by perara** [comment]
Yes, This is because I've implemented it using the HA translations system. However this is not supported by custom_components.

I'll check with the HA peeps if there is any way around this, while keeping the translations.
